cmd/nerdctl/create_linux_test.go

```diff
@@ -106,3 +110,47 @@ func TestCreateWithMACAddress(t *testing.T) {
 		}
 	}
 }
+
+func TestCreateWithTerminal(t *testing.T) {
+
+	if runtime.GOOS == "windows" {
+		t.Skip("buildkit is not enabled on windows, this feature may work on windows.")
+	}
+	testutil.DockerIncompatible(t)
```

Comment:
Why incompatible?

Reply:
> Why incompatible?

using `-d` will use the json-file log driver and cause the windows test fail